src/main/java/kr/codesqaud/cafe/domain/article/repository/ArticleJdbcImpl.java

```diff
@@ -0,0 +1,42 @@
+package kr.codesqaud.cafe.domain.article.repository;
+
+import kr.codesqaud.cafe.domain.article.Article;
+import kr.codesqaud.cafe.domain.user.User;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class ArticleJdbcImpl implements ArticleRepository {
```

Comment:
인터페이스를 구현하신 점 좋습니다. 👍 다만, 데이터 엑세스 계층은 Impl 을 suffix 로 가져가기보다는 데이터베이스 접근 기술로 구분하시는 건 어떨까요?

```suggestion
public class ArticleJdbcRepository implements ArticleRepository {
```

이전에 작성하신 메모리를 저장소로 사용한 레퍼지토리는

```java
public class ArticleMemoryRepository implements ArticleRepository {
}
```

이렇게요.